src/models/GOKU.jl

```diff
@@ -5,8 +5,21 @@
 
 struct GOKU <: LatentDE end
 
+@doc raw"""
+    apply_feature_extractor(encoder, x)
+Converts a batch of the initial high-dimensional data into lower-dimensional data (i.e. extracts features)
+# Arguments
+encoder: Has type Encoder{GOKU} or Encoder{LatentODE}\
+x: Initial data, has size (seq_len,), x[1] has size (input_dim, batch_size)
+"""
 apply_feature_extractor(encoder::Encoder{GOKU}, x) = encoder.feature_extractor.(x)
 
+@doc raw"""
+    apply_pattern_extractor(encoder::Encoder{GOKU}, fe_out)
+Passes features in time series through RNNs, returning a tuple containing patterns for the initial state and the parameters, respectively.
```

Comment:
They are not necessarily RNNs. As in the previous case I would say something like

Passes extracted features through the pattern_extractor layer contained in the `encoder`, returning a tuple containing corresponding to the initial latent states and parameters, respectively.

Reply:
Okay, will replace description with what you have, deleting "containing"